src/templates/common/mediaImage/index.tsx

```diff
@@ -63,6 +63,10 @@ export const MediaImage = (
     return null
   }
 
+  if (!props.alt) {
+    return null
+  }
+
```

Comment:
This is to mirror [what's happening on content-build](https://github.com/department-of-veterans-affairs/content-build/blob/main/src/site/components/image.drupal.liquid#L1-L1).

Alt text is a required field in Drupal, so this shouldn't happen, but a bit o' defensive programming here makes certain that we're not displaying images without alt text.